### PR TITLE
Login

### PR DESCRIPTION
**Pull request by danielwatson6**

### CODE REVIEW

**Comment by jaisal1024** [comment]
You commented in ./src/server/routes/login.py line 19 that we need to implement a remember me function. But then you added the login cookies, so isn't that done?

File: src/scripts/add_user.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from getpass import getpass
```
Comment (jaisal1024) [inline src/scripts/add_user.py]: we should probably adapt this later on into an interface for users to use.

**Comment by danielwatson6** [comment]
> You commented in ./src/server/routes/login.py line 19 that we need to implement a remember me function. But then you added the login cookies, so isn't that done?

The cookies are deleted when the browser is closed. A "remember me" box can be used to make the cookies persist, but this is a tradeoff between UX and security

File: src/scripts/add_user.py (hunk 1 above)
Comment (danielwatson6) [reply src/scripts/add_user.py]: Unsure. The software is intended for admins. So then how to block any undesired party from having access to that registration page?

**Pull request merged**